src/strategies/planet-finance/index.ts

```diff
@@ -155,6 +211,25 @@ export async function strategy(
 
   const contractAquaBalance = erc20Result.lpAquaBal.toString();
 
+  //AQUA-GAMMA
+  erc20Multi.call(
+    'aquaGammaTotalSupply',
+    aquaGammaLpTokenAddress,
+    'totalSupply'
+  );
+
+  erc20Multi.call('lpAquaBal', aquaAddress, 'balanceOf', [
+    aquaGammaLpTokenAddress
+  ]);
+
+  erc20Result = await erc20Multi.execute();
```

Comment:
Hello @ash-greninja-poke These changes are adding many requests, we have a limit of 5 requests 
https://github.com/snapshot-labs/snapshot-strategies#checklist-for-adding-a-new-strategy

Can you check if there is a way to get all these in a single request